src/redux/features/orders/ordersSlice.js

```diff
@@ -1,5 +1,6 @@
 // jshint esversion:9
 
+// eslint-disable-next-line no-unused-vars
```

Comment:
I recommended add this in that specific case. Don't add it in every case, it's not a good practice